daphne/src/lib.rs

```diff
@@ -421,6 +470,87 @@ impl DapTaskConfig {
     pub(crate) fn truncate_time(&self, time: Time) -> Time {
         time - (time % self.time_precision)
     }
+
```

Comment:
Reviewer note: This replaces https://github.com/cloudflare/daphne/pull/133/files#diff-e9132ff085252548c520081f56624153a00c9a0b0310982ae9a9aae165e6a9f3L606-R742